zerver/models/streams.py

```diff
@@ -187,6 +187,9 @@ def is_history_realm_public(self) -> bool:
     def is_history_public_to_subscribers(self) -> bool:
         return self.history_public_to_subscribers
 
+    def is_support_stream(self) -> bool:
+        return self.stream_topic_access_group.name != SystemGroups.EVERYONE
+
```

Comment:
Accessing `stream_topic_access_group.name` is leading to an inflated number of database queries. Unfortunately, I can't find a way around.